attendance/API_HR/c-hr.php

```diff
@@ -2092,7 +2092,17 @@ public static function updateLeaveStatus($leaveid, $newstatus, $messagetouser) {
             $applied_on = $leaveDetails['applied_on'];
             $reason = $leaveDetails['reason'];
 
-            self::changeLeaveStatus($leaveid, $newstatus);
+            $changeLeaveStatus = self::changeLeaveStatus($leaveid, $newstatus);
+            if( strtolower($leaveDetails['leave_type']) == 'restricted' ){
+                if( $changeLeaveStatus ){
+                    $updatedLeaveDetails = self::getLeaveDetails($leaveid);
+                    if( strtolower($updatedLeaveDetails['status']) == 'approved' ){
+                        $entry_time = "10:30 AM";
```

Comment:
can define these timing as constant, so that default entry and default exit time will be used from a single place.